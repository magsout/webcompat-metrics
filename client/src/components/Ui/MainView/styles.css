/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
:root {
  --fontSize: var(--fontSize);
  --marginLeftComponent: var(--widthMenu);
}

.component {
  font-size: var(--fontSize);
  overflow-x: hidden;
  flex: 1;
  padding: 1em;
}

@media all (min-width: 42.5em) {
  .component {
    margin: 0 0 0 var(--marginLeftComponent);
  }
}
