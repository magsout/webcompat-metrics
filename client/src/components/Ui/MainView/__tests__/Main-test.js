/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import MainView from "../";

it("renders MainView default correctly", () => {
  const tree = renderer
    .create(
      <MainView>
        <span>content</span>
      </MainView>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
