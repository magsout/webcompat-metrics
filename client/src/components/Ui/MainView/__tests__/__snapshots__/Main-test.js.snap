// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MainView default correctly 1`] = `
<main
  className="component"
  role="main"
>
  <span>
    content
  </span>
</main>
`;
